docs/docusaurus/docs/contributing/application_integration_support.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Integration support policy
+---
+
+For production environments, GX recommends using the integrations defined as GX-supported. 
+
+GX uses libraries such as Pandas, Spark, and SQLAlchemy to integrate with a wide range of data sources. This also allows you to deploy GX with Community-supported integrations.
+
+## Support categories
+
+The following are the levels of support provided by GX:
+
+- GX-supported - integrations are tested throughout the development lifecycle, and actively maintained and updated when new versions of GX Cloud or GX OSS are released.
+
+- Community-supported - integrations are implemented by community members. They are not tested or updated by GX, and GX is not responsible for ensuring reliability or compatibility.
```

Comment:
These integrations are are not just implemented by community members. Some of these were created by GX. We also test and update some of them. We've also seen successful implementations of some of these. I'm ok with the last piece around responsible for reliability or compatibility.